docs/products/databases/managed-databases/guides/updates-and-maintenance/index.md

```diff
@@ -0,0 +1,37 @@
+---
+author:
+  name: Linode
+  email: docs@linode.com
+title: "Automatic Updates and Maintenance Windows"
+description: "Managed databases are automatically updated on a weekly or monthly basis. Learn how to manage these maintenance windows in the Cloud Manager"
+date: 2022-04-15
+modified: 2022-04-15
+---
+
+As part of the Managed Database service, the database cluster is updated regularly. This includes security updates and patches for the underlying operating system. These updates occur on a user-configurable *maintenance window*. By default, the maintenance window is set to start *every week* on *Sunday* at *20:00 UTC* and lasts for 3 hours.
```

Comment:
This isn't accurate. The default maintenance window is weekly, but the default day/time are random-ish depending on when the database is created. Maybe we could just say a default maintenance window is pre-set, but is changeable at any time by the customer? The duration of 3 hrs is correct.

Reply:
Going to leave this as-is, since this accurately describes the current behavior.